shared/test/fixtures/vcr/webpurify/find_potential_profanity.yml

```diff
@@ -0,0 +1,189 @@
+---
```

Comment:
Unfortunately, our framework is set up to store these under `shared/test/` rather than `lib/test/`.